private/syntax-replacement.rkt

```diff
@@ -295,6 +301,35 @@
        (check-equal? (syntax-replacement-render replacement) expected)])))
 
 
+(define (syntax-replacement-preserves-free-identifiers? replacement)
+  (match replacement
+    [(syntax-replacement #:original-syntax orig
+                         #:new-syntax new
+                         #:introduction-scope intro)
+     ((stx-free-ids=?/ignore
+       (ignore-id? intro (list #'SPACE #'NEWLINE #'ORIGINAL-GAP #'ORIGINAL-SPLICE)))
```

Comment:
Does `stx-free-ids=?/ignore` need to be curried? Why not pass in all the arguments at once?